xbmc/interfaces/legacy/AddonUtils.h

```diff
@@ -68,6 +69,9 @@ namespace XBMCAddonUtils
   };
 
 #define LOCKGUI XBMCAddonUtils::GuiLock __gl
+#define LOCKGUIIF(cond) std::unique_ptr<XBMCAddonUtils::GuiLock> __gl; \
```

Comment:
Afaik all our python hooks call unblock_threads before calling into Kodi itself, trying to lock the Gil again here might be problematic. Testing will show i guess